### ISSUE TITLE
Peakrdl_html cannot have 2 consecutive targets because of OUTDIR

### ISSUE THREAD

**Issue by Risto97** [HEP-SoC/SoCMake#43]
If both IP libraries are defined in same project, they share the same BINARY_DIR.

If we create 2 html targets they will conflict:
change:
```
        set(OUTDIR ${BINARY_DIR}/html)
```

to:

```
        set(OUTDIR ${BINARY_DIR}/${IP_LIB}_html)
```

**Comment by Risto97** [comment]
Also missing `ip_assume_last()`

**Comment by Risto97** [comment]
Resolved by #89
